spec/about_spec.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './spec_helper'
+
+describe WulfApp do
+    it "shows about page" do
+        visit "/about"
```

Comment:
There's no validation in this test, but it's a static page, so it either crashes hard or works perfectly fine.